doc/release/release_notes.rst

```diff
@@ -4,6 +4,28 @@
 Release Notes
 *************
 
+.. release:: Upcoming
+
+    .. change:: changed
+
+        Added compatibility with python 3.6, 3.7 and 3.8.
+
+    .. change:: fixed
+
+        Fixed :class:`wiz.graph.Resolver` to ensure that conflicted nodes are
+        always sorted in ascending order of distance from the :attr:`root
+        <Graph.ROOT>` level of the graph.
+
+        Previously, conflicting nodes wouldn't be sorted properly when new
+        packages are added to the graph during the conflict resolution process.
```

Comment:
resulting in what? How did wiz ever work if this didn't work in the first place?

Reply:
Well it's a particular case when a new package is added during the conflict resolution process. See scenario 7 and 10 in the integration tests:

```
    Root
     |
     |--(A<=0.3.0): A==0.3.0
     |
     |--(B): B==0.1.0
     |   |
     |   `--(A !=0.3.0): A==1.0.0
     |
     `--(C): C==0.1.0
         |
         `--(A <1): A==0.9.0

    Expected: C==0.1.0, B==0.1.0, A==0.2.0
```

These test passed correctly in Python 2.7 because somehow the [conflict list concatenation](https://github.com/themill/wiz/blob/master/source/wiz/graph.py#L619) step produce surprisingly constant results...

Reply:
Release notes updated